act-rules-format.md

```diff
@@ -170,16 +170,26 @@ Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em>
 4. the conformance level associated with the accessibility requirement, if one exists.
 
 
-### Outcome Mapping ### {#outcome-mapping}
+### Mapping Conformance Requirements ### {#mapping-conformance-requirements}
+
+An ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. 
+
+A conformance requirement mapped in a rule: 
+
+- is <dfn>not satisfied</dfn> when one or more outcomes for a test target is `failed`, and 
```

Comment:
I think change `test target` to `test subject` here since the subject can have multiple outcomes.